### DISCUSSION TITLE
post/2021-11-24-dapr-operator/

### DISCUSSION THREAD

**Discussion by giscus** [zcong1993/blog#47] [Announcements]
# post/2021-11-24-dapr-operator/

本文介绍 Dapr Operator 相关源码.

https://blog.cong.moe/post/2021-11-24-dapr-operator/

**Comment by realzcong** [comment]
这是一条 https://giscus.app/zh-CN 测试评论

**Reply by zcong1993** [reply]
这是一条回复测试

**Reply by realzcong** [reply]
这是一条嵌套回复测试 :apple:
